test/tests/providers/global/platform/atlas/alerting-rules/kube-state-metrics.rules.test.yml

```diff
@@ -87,7 +87,7 @@ tests:
               provider: "capa"
               pipeline: "testing"
             exp_annotations:
-              description: "KubeStateMetrics () is down."
```

Comment:
How did the tests pass? The description was already `KubeStateMetrics is down.` before this PR, right? These tests should have failed?

Reply:
I think that this alert is in the promtool ignore thingy